htdocs/core/modules/member/doc/doc_generic_member_odt.class.php

```diff
@@ -413,7 +414,6 @@ public function write_file($object, $outputlangs, $srctemplatepath, $mode = 'mem
 		return -1;
 	}
 
-	// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps
```

Comment:
Removing this generate errors with phpcs run by Travis. This is the expected behaviour.
Do yo know why it does not report error with phpcs run by pre-commit ?

Reply:
I have made some changes to the phpcs github action but I want to test it better.
The core issue is that phpcs returns exit code 1 even for errors, so I updated the test in the github action but did not succeed  yet in adding a testcase for the github action to validate it.

Reply:
I validated the updated php-cs pre-commit hook with this PR (I attached the updated configuration before fixing the issues).